css/style.css

```diff
@@ -0,0 +1,2451 @@
+  }
+
+  /* HOVER effects for vegburger menu on med: */
+    #veggie-burger:hover {
+      background-color: #ffff00;
+      transition: 0.3s;
+    }
+
+    #veggie-burger:hover .veggie-patty {
+      background-color: #000000;
+      transition: 0.3s;
+    }
+
+  /* FOCUS effects for vegburger menu on med: */
+    #veggie-burger:focus {
+      outline: 3px solid #ffff00;
+      outline-offset: -5px;
+    }
+
+  /* righthand icons */
+  #search, #language, #user, #developers {
+    height: 3.7rem;
+    background-color: #2a2a2a;
+    border: none;
+    padding: 0 1.25rem
+  }
+
+  /* find a better way to do this */
+  #language {
+    padding-top: 8px;
+  }
+
+  #user, #developers {
+    font-size: 10px;
+  }
+
+  #developers {
+    background-color: #ffff00;
+    color: #2a2a2a;
+  }
+
+  /* text in "sign up" button */
+  #user .sign-text {
+    padding: 0 5px;
+  }
+
+  /* fontawesome icons */
+  #search .fas, #user .fas, #language .fas {
+    color: #CCCCCC;
+    line-height: 1.6;
+    display: inline;
+  }
+
+  /* all nav text incl buttons: */
+  .nav-text {
+    text-transform: uppercase;
+    color: #CCCCCC;
+    font-size: 10.5px;
+    font-family: 'Open Sans', sans-serif;
+    letter-spacing: 1px;
+  }
+
+  /* make the bottom border already exist so that it doesn't bump up the text */
+  #nav-right button {
+    border-bottom: 5px solid #2a2a2a;
+    border-top: 5px solid #2a2a2a;
+  }
+  #nav-right #developers {
+    border-bottom: none;
+    border-top: none;
+  }
+
+  /* HOVER effect for righthand buttons: yellow line */
+  #nav-right button:hover {
+    border-bottom: 5px solid #ffff00;
+    transition: 0.3s;
+  }
+
+  /* HOVER effect for righthand buttons: white font */
+  #search:hover .fas,
+  #search:hover,
+  #language:hover .fas,
+  #user:hover .fas,
+  #user:hover .sign-text {
+    color: #ffffff;
+  }
+
+  /* FOCUS effect for righthand buttons: yellow outline */
+  #nav-right button:focus,
+  #language:focus {
+    outline: 3px solid #ffff00;
+    outline-offset: -5px;
+  }
+
+  /* FOCUS effect for righthand buttons: white font */
+    #nav-right button:focus .fas,
+    #nav-right button:focus span,
+    #language:focus .fas {
+      color: #ffffff;
+    }
+
+  /* HOVER effect for developers button: make glow blue and yellow */
+  #nav-right #developers:hover {
+    color: #000000;
+    border: none;
+    animation: glow 3s linear infinite;
+  }
+
+  /* FOCUS effect for developers button: blue outline */
+  #nav-right #developers:focus {
+    outline: 3px solid #1EB8F4;
+    outline-offset: -5px;
+    color: #000000;
+  }
+
+  /* nav dropdown (competitive and language) */
+  .main-nav-links, .dropbtn {
+    display: inline-block;
+  }
+
+  /* make it invisible til hover but set the styling */
+  .dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #2a2a2a;
+    min-width: 180px;
+    max-width: 180px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    right: 210px;
```

Comment:
change "right: 210px" to "right: 160px" to re-align the language dropdown, please!

Reply:
changed